crates/cheatnet/src/execution/cheatable_syscall_handler.rs

```diff
@@ -93,13 +93,12 @@ impl HintProcessorLogic for CheatableSyscallHandler<'_> {
             }
         }
 
-        match contract_print(vm, maybe_extended_hint) {
-            PrintingResult::Printed => return Ok(()),
-            PrintingResult::Passed => (),
-            PrintingResult::Err(err) => return Err(err),
-        }
-        self.syscall_handler
-            .execute_hint(vm, exec_scopes, hint_data, constants)
+        ContractSyscallHandler::wrap(&mut self.syscall_handler).execute_hint(
+            vm,
+            exec_scopes,
+            hint_data,
+            constants,
+        )
```

Comment:
For the consistency with `TestExecutionSyscallHandler` i would keep `CheatableSyscallHandler` as a field of `ContractSyscallHandler`

Reply:
[Address review suggestions](https://github.com/foundry-rs/starknet-foundry/pull/924/commits/a9245c7ef18afb28f9dee6d75699d9b040e7b802)